cardano-api/src/Cardano/Api/Fees.hs

```diff
@@ -444,7 +447,7 @@ instance Error TransactionValidityIntervalError where
 
         | otherwise
         = 0 -- This should be impossible.
-
+  displayError (TransactionValidityBasicFailure (Alonzo.UnknownTxIns _)) = "Unknown txins present" -- TODO More information error message
```

Comment:
How can I better display this error?

Reply:
@JaredCorduan ?

Reply:
See that this was already updated (but reverted), so you could reuse things there: https://github.com/input-output-hk/cardano-node/pull/3410/files#diff-12e0bb9f3db22ac71c93ed3932f54c91a6f6bf7243afaa29ccd7ecb3657e6069L429